react-vis-force/build.boot

```diff
@@ -0,0 +1,32 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
In addition to `deps-cljs :require` package must also depend on `cljsjs/react` if React is needed, so add a dependency here.